### DISCUSSION TITLE
Payments: Mollie integration roadmap

### DISCUSSION THREAD

**Discussion by Juleszz** [saleor/saleor#8149] [General] [answered]
Hey there, 

I just saw that Mollie will be integrated soon and I just wanted to check if you have a rough idea of when this will be implemented. I'm currently working on the checkout but if it will be released soon it's worth waiting as it's a good option to integrate Paypal, too.

Thanks!

**Comment by karolkielecki** [comment, marked as answer]
@Juleszz Hey, we don't have ETA for now. I'll keep you updated if there is any news.

**Reply by toastedtoast** [reply]
Hi @karolkielecki 

I know it's annoying to ask again but I am also pretty interested in using mollie over i.e. Braintree. 
As mollie is not mentioned on the roadmap which roughly lays out the expected features for the next 2Qs I wonder if it is out of scope for this timeframe or just not mentioned. 
Also as 2 month have passed since @Juleszz has posted his Question, maybe there is a ETA now. 😉

Kind Regards,
Marco

**Reply by karolkielecki** [reply]
Hi @toastedtoast. No changes here -- there is no ETA. We have other priorities for now - Saleor 3.0 and 3.1.

**Reply by JulianAtTheFrontend** [reply]
We just integrated Mollie in a different way and it's working pretty great.

Just checkout [rewhite-climbing.de](www.rewhite-climbing.de), put some items to your bag and proceed to checkout. At one point you'll be forwarded to Mollie and once you come back (i know you cant test that right now without actually paying) the checkout is completed.

It's not 100% perfect but for now the best option and we'll keep it updated over time.

**Reply by toastedtoast** [reply]
Hi @JulianAtTheFrontend
First, very nice shop! Looks amazing. 👏

I see that you are proxing the payment using some google function I assume to hide the mollie credentials, right? + You are using the Saleor Payment Dummy. This is required, to "complete" the payment for that particular order in saleor then, right?

Is there any chance to have a look at the implementation of the cloud function?

**Reply by JulianAtTheFrontend** [reply]
@toastedtoast  thanks! 
Yes, I am using google cloud functions to secure the Mollie credentials (it will also throw an error otherwise). The Payment Dummy is a good way to tell Saleor that the payment is handled and I can then choose manually wether a customer is redirected to Mollie or if the checkout should be completed directly - e.g. when a B2B customers chose to pay later.

Just contact me via julian@the-frontend.de and I can send you some code from the checkout. I'm using Vue but the logic is pretty much straight forward JS.
